incubator/acs-engine-autoscaler/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+To verify that acs-engine-autoscaler has started, run:
+
+  kubectl --namespace={{ .Release.Namespace }} get pods -l "app={{ template "fullname" . }}"
+
```

Comment:
As this chart requires credentials to run, you should output an error if they're missing. Check out https://github.com/kubernetes/charts/blob/master/stable/sumokube/templates/NOTES.txt for one pattern. No point in a user checking pods if core config is missing.